conf/settings.py

```diff
@@ -145,18 +147,15 @@
 # to load the internationalization machinery.
 USE_I18N = True
 
-# If you set this to False, Django will not format dates, numbers and
-# calendars according to the current locale.
-USE_L10N = True
-
 # If you set this to False, Django will not use timezone-aware datetimes.
 USE_TZ = True
 
 # Type of automatic primary key, introduced in Django 3.2
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
 # URL prefix to use, please see documentation for more details
-URL_PREFIX = "__PATH_URL__"
+# WARNING: this must be without trailing slash (this is why we set __PATH2__ (cf. loaded settings in install and upgrade))
+URL_PREFIX = "__PATH2__"
```

Comment:
You could name it "path_no_slash" for readability

Reply:
Oooh nice, good idea!